### DISCUSSION TITLE
Welcome to protractor-cucumber-typescript Discussions!

### DISCUSSION THREAD

**Discussion by igniteram** [igniteram/protractor-cucumber-typescript#80] [General]
## 👋 Welcome!
  I am starting a discussion page for this framework, I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas about new automation features.
  * Engage with other members who use this framework.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sachinkes** [comment]
Hi ,

I am new to protractor and I got yours git project on web .

I am referring your project but when i am creating a new feature file or a step definitions its not working

can you plz help ASAP
